docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,225 @@
+- `soci-snapshotter-grpc`: the daemon (a containerd snapshotter plugin) used for lazy loading.
 
-## Start a local ORAS registry
-Most OCI registries do not currently support the "referrers" feature.
-SOCI relies on this feature to link its indices and manifests.  The [ORAS
-project](https://oras.land/) tries to fill this gap; in their words:
+Many `soci` CLI commands need to be run as `sudo`, because the metadata is saved
+in directories that a non-root user often does not have access to.
 
-> Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
+Currently to get the binaries, we need to build the project from source after cloing the repo:
 
-[The longer term solution for this issue is using the OCI Reference
-Types](https://github.com/awslabs/soci-snapshotter/issues/42).  For now we are
-relying on the ORAS project for the referrers feature.
+```shell
+$ git clone https://github.com/awslabs/soci-snapshotter.git
+$ cd soci-snapshotter
+$ make
+```
 
-Given this we want to start our registry using an OCI image from the ORAS 
-project.  We will run it on port 5000 as an HTTP (**NOT** HTTPS) server without
-certificates.  To do this we will use Docker:
+This builds the project binaries into the `./out` directory. You can install them
+to a `PATH` directory (`/usr/local/bin`) with:
 
-```
-docker run -d -p 5000:5000 --restart=always --name registry ghcr.io/oci-playground/registry:v3.0.0-alpha.1
+```shell
+$ sudo make install
 ```
 
-Since this is just a locally run HTTP server we can interact with
-it using standard `curl` commands. To confirm this is running lets
-use the following commands:
+## Push an image to your registry
 
-```
-docker ps
-```
+In this document we will use `rabbitmq` from DockerHub `docker.io/library/rabbitmq:latest`.
 
-which should yield a running container at 0.0.0.0:5000 for the ORAS image. Now
-to check that we can interact with the server run the following:
-```
-curl http://localhost:5000/v2/_catalog
+> You need to change `<AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com/rabbitmq` to your own registry.
+> The platform tag might be different depending on your machine.
+
+```shell
+$ sudo ctr i pull docker.io/library/rabbitmq:latest
+$ sudo ctr i tag docker.io/library/rabbitmq:latest <AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com/rabbitmq:latest
+$ sudo ctr i push --platform linux/amd64 <AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com/rabbitmq:latest
 ```
-This should return with `{"repositories":[]}` given that we haven't put anything
-in there yet
 
-## Push an image to the registry using ctr
-First things first, let's choose our favorite image.  For the purpose of this
-document we will use the latest image of rabbitmq from DockerHub, aptly named
-`docker.io/library/rabbitmq:latest`.
+After this step, please check your registry to confirm the image is pushed to it.
```

Comment:
Can we keep an example command for this?

Reply:
fixed